README.md

```diff
@@ -21,10 +24,11 @@ jobs:
       - uses: gleam-lang/setup-erlang@v1.1.2
         with:
           otp-version: 23.2
+          rebar3-version: 3.16.1
       - run: rebar3 eunit
 ```
 
-### Matrix example
+### Matrix example with Rebar3 version that can be used with the OTP version
```

Comment:
Can we split these examples into their own? I would like each one to show one feature and I think it is a little less clear when combining them.